### DISCUSSION TITLE
Install embassyOS on my spare laptop

### DISCUSSION THREAD

**Discussion by grantathon** [Start9Labs/start-os#2041] [Q&A] [answered]
I was wondering if it's possible to install the OS directly on my laptop, and if so how?

**Comment by chrisguida** [comment, marked as answer]
Yes, with a very large caveat: 0.3.3 has not been released yet and is not stable, so things may break and we don't have the resources to provide support in such circumstances, so you're on your own.

If you're just tinkering and/or want to help us test, you can click "actions" and look for the iso produced by the "PureOS Base ISO" workflow (assuming your laptop has x86 architecture).

Otherwise please hold off until we release 0.3.3 :)

**Comment by grantathon** [comment]
Alright cool, I'll wait for the 0.3.3 release!

**Comment by chrisguida** [comment]
You're in luck haha

https://github.com/Start9Labs/embassy-os/releases/tag/v0.3.3
